packages/collector/test/tracing/cloud/aws-sdk/v2/sqs/test.js

```diff
@@ -28,7 +28,7 @@ const { verifyHttpRootEntry, verifyHttpExit } = require('@instana/core/test/test
 const defaultPrefix = 'https://sqs.us-east-2.amazonaws.com/410797082306/';
 const queueUrlPrefix = process.env.SQS_QUEUE_URL_PREFIX || defaultPrefix;
 
-let queueName = 'team_nodejs';
+let queueName = 'nodejs-team';
```

Comment:
A queue with this name actually existed in our AWS account, while `team_nodejs` did not exist.

Reply:
As long as the clean-aws.js script still works, I am fine with this change 👍